provider/pkg/provider/provider.go

```diff
@@ -2464,7 +2465,7 @@ func (k *kubeProvider) Delete(ctx context.Context, req *pulumirpc.DeleteRequest)
 
 	config := await.DeleteConfig{
 		ProviderConfig: await.ProviderConfig{
-			Context:           k.canceler.context, // TODO: should this just be ctx from the args?
```

Comment:
Removing the above comment because it is 4 years old, and it seems clear that the code is correct and is consistent with the other provider methods (`Create`, `Update`).